frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -70,9 +68,6 @@ const ImageStreamTagDropdown: React.FC<{
           setFieldValue(`${fieldPrefix}isi.tag`, selectedTag);
           setFieldValue(`${fieldPrefix}isi.ports`, ports);
           setFieldValue(`${fieldPrefix}image.ports`, ports);
-          formType !== 'edit' &&
-            resourceName !== name &&
-            setFieldValue(`${fieldPrefix}name`, getSuggestedName(name));
```

Comment:
@Lucifergene removing this the name will not be auto generated in the add forms as well. Also instead of removing this you should try to find out why the name is getting changed even when the statement allows generating a name only when its not an edit form.

Reply:
Okay, I understand
Thanks

Reply:
I found out that the `formType` is **undefined**.
Therefore I am checking the nature of the form (create/edit) from the URL parameters as seen in some other files.
The logic is that if the URL param contains `name` param, it's an `edit` form otherwise it's a `create` form.

Reply:
@Lucifergene you can also add `formType: 'edit'` to edit deployment form initial values instead of doing this to keep it consistent with other edit forms. Also check if this issue only exists in this form or some other form as well.